ddtrace/appsec/processor.py

```diff
@@ -67,26 +122,82 @@ def __attrs_post_init__(self):
                 # Partial of DDAS-0005-00
                 log.warning("[DDAS-0005-00] WAF initialization failed")
                 raise
+        for address in self._ddwaf.required_data:
+            self._mark_needed(address)
```

Comment:
is it possible for `libddwaf` to give us addresses not defined in this doc?

what would happen in that case?

Reply:
if that were the case nothing would happen. Only the implemented addresses will be checked and used